manifest.yaml

```diff
@@ -5,56 +5,60 @@ metadata:
     control-plane: registry-creds-controller
   name: registry-creds-system
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This file should be generated, not edited manually.

The change needs to happen in the codegen